flink-runtime/src/main/java/org/apache/flink/runtime/io/network/api/serialization/SpanningWrapper.java

```diff
@@ -43,15 +46,18 @@
 
 import static java.lang.Math.max;
 import static java.lang.Math.min;
+import static org.apache.flink.core.memory.MemorySegmentFactory.wrapCopy;
+import static org.apache.flink.core.memory.MemorySegmentFactory.wrapInt;
 import static org.apache.flink.runtime.io.network.api.serialization.NonSpanningWrapper.singleBufferIterator;
 import static org.apache.flink.runtime.io.network.api.serialization.SpillingAdaptiveSpanningRecordDeserializer.LENGTH_BYTES;
+import static org.apache.flink.util.CloseableIterator.empty;
 import static org.apache.flink.util.FileUtils.writeCompletely;
 import static org.apache.flink.util.IOUtils.closeAllQuietly;
 
 final class SpanningWrapper implements BufferWrapper {
 
-	private static final int THRESHOLD_FOR_SPILLING = 5 * 1024 * 1024; // 5 MiBytes
-	private static final int FILE_BUFFER_SIZE = 2 * 1024 * 1024;
+	private static final int DEFAULT_THRESHOLD_FOR_SPILLING = 5 * 1024 * 1024; // 5 MiBytes
+	private static final int DEFAULT_FILE_BUFFER_SIZE = 2 * 1024 * 1024;
```

Comment:
I agree that it could be less but it's the same constant used for "normal" reading of "spill" file.
I don't think it should be a different number of that we should change it in this PR (and without some experimentation).